UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/IAzureBlobStorageClient.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.Common.Models.SalesCatalogue;
+
+namespace UKHO.ExchangeSetService.Common.Helpers
+{
+    public interface IAzureBlobStorageClient
+    {
+        Task<bool> StoreScsResponseAsync(string containerName, string batchId, SalesCatalogueResponse salesCatalogueResponse, CancellationToken cancellationToken);
+    }
+}
```

Comment:
Could be StoreSalesCatalogueServiceResponseAsync

Reply:
Done